apps/browser-extension-wallet/src/stores/slices/wallet-activities-slice.ts

```diff
@@ -105,56 +126,31 @@ const getWalletActivitiesObservable = async ({
       cardanoCoin
     });
 
+    const finalizeTransaction = (transformedTx: Omit<AssetActivityItemProps, 'onClick'>) => ({
```

Comment:
```suggestion
    const extendWithClickHandler = (transformedTx: Omit<AssetActivityItemProps, 'onClick'>) => ({
```

or at least `finalizeTransactionEntryData`

`finalizeTransaction` sounds to me like it would be submiting the transaction 😉